gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -458,6 +458,10 @@ def main(args):
                 cochran_mantel_haenszel_test=res.cmh_ht.ht()[
                     ht.key
                 ].cochran_mantel_haenszel_test,
+                data_type=hl.case()
+                .when(~hl.is_defined(ht.genomes_freq), "exomes")
+                .when(~hl.is_defined(ht.exomes_freq), "genomes")
```

Comment:
So we also need to make sure that the variant is not exclusive to one of the groups in `POPS_TO_REMOVE_FOR_FAF`. I think we can do this by just checking if `genomes_faf` or `exomes_faf` is defined instead of `*_freq`...I'm not totally clear where we want this flag though so double checking with @ch-kr

Reply:
This also makes it a top level annotation -- I would think wed want the `data_type` annotation nested somewhere for the release, or I guess we could renamed it there to something like `joint_metric_data_type` if we want it top level, that just feels confusing though.

Reply:
>So we also need to make sure that the variant is not exclusive to one of the groups in POPS_TO_REMOVE_FOR_FAF.  I think we can do this by just checking if genomes_faf or exomes_faf is defined instead of *_freq

yes, please update to checking if faf is defined rather than just the frequency

>I'm not totally clear where we want this flag though so double checking with @ch-kr

I agree this should be nested on the release HTs (probably in the fafmax struct). for the joint FAF resource, I actually kind of like this as a top level annotation if renamed to `joint_metric_data_type`, since it explains which data type is relevant for each joint metric on that row. that said, I don't have strong feelings about this

Reply:
That makes sense to me too -- top level here as joint_metric_data_type and I'll nest it in the release joint_fafmax as `data_type`

Reply:
so, we want to check if genomes_faf and exomes_faf are defined, or they have non-0 values?